docs/generated/packages/gradle/documents/overview.md

```diff
@@ -14,21 +14,50 @@ Nx adds the following features to your workspace:
 - [Run only tasks affected by a PR](/ci/features/affected)
 - [Interactively explore your workspace](/features/explore-graph)
 
+{% callout type="info" title="Java Compatibility" %}
+This plugin has a minimum compatible of Java 17.
+We are open to discussing to lowering this minimum so please reach out to us on [Twitter/X](https://x.com/NxDevTools), [Bluesky](https://bsky.app/profile/nx.dev), [LinkedIn](https://www.linkedin.com/company/nrwl/), or join our [Discord](https://go.nx.dev/community)!
+{% /callout %}
+
+## Install Nx without NodeJS
+
+If you want to use Nx without Node, you can install Nx via Homebrew with the following command:
+
+```shell {% skipRescope=true %}
+brew tap nrwl/nx && brew install nx
```

Comment:
personally, I'd suggest leaving as two separate commands but understand the point of a 1 liner.

might be worth calling out for windows folks there's plans to have the same functionality for windows users down the road.